translations/de.ts

```diff
@@ -1037,7 +1041,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <location filename="../src/widget/form/settings/generalsettings.ui" line="119"/>
         <source>qTox will start minimized in tray.</source>
         <comment>toolTip for Start in tray setting</comment>
-        <translation>qTox minimiert starten.</translation>
+        <translation>qTox wird minimiert starten.</translation>
```

Comment:
Ach so, jetzt verstehe ich den Satz. Genau übersetzt würde es dann "qTox wird in der Startleiste minimiert starten." heißen. Dazu sage ich erstmal nix. Die Übersetzung wird eh von mal zu mal besser. Yay, es besteht Hoffnung ... :)